script/Arcueid/Day 12/12_00_ARC12_2.txt

```diff
@@ -684,13 +685,13 @@ My body temperature is dropping.
 -- Page 70, Offset 30728.
 -- 「おまえ<が|・>殺された借り、たしかに返却した。もはや思い残す事もないだろう。
 // Reading these 2 lines as addressed to SHIKI not Shiki
-"It would appear that you have returned the favor of being killed.
- You must have no more regrets now.
+"You have most certainly gotten back at him for killing you.
```

Comment:
Kinda prefer the 'repayment in kind' clarity of the original.

Reply:
I don't think it's crazy for 確かに, but maybe it's a british bent? "most certainly" just seems a little weird to me. Maybe someone else can weigh in.

Reply:
Yeah, actually "most certainly" isn't quite right, though that's me thinking more about the Japanese nuance. I am all for changing it, I just don't think "it would appear" is right either. 

I think its almost closer to "as promised" in this instance.

Reply:
> With this, you have repaid him in kind for having killed you. Surely you have no regrets now.

?